src/answers-umd.js

```diff
@@ -86,8 +86,9 @@ class Answers {
      * A reference to the formatRichText function.
      * @type {Function}
      */
-    this.formatRichText = (markdown, eventOptionsFieldName, targetConfig, truncatedLength) =>
-      RichTextFormatter.format(markdown, eventOptionsFieldName, targetConfig, truncatedLength);
+    this.formatRichText = (markdown, eventOptionsFieldName, targetConfig, truncatedLength, truncatedSuffix) =>
```

Comment:
if this is just passing through the arguments, I wonder if it would make sense to do this? 
`this.formatRichText = RichTextFormatter.format`?

Reply:
Ah ok I just tried it and it changes the `this` context of RichTextFormatter.format from the RichTextFormatter instance to I think the answers-umd instance
![image](https://user-images.githubusercontent.com/23005393/162469044-3adf8ce5-dff6-4010-bdbe-ba410e6739b2.png)

This could be pretty easily fixed by pulling out the `this._urlTransformer` and `this._generatePluginName` methods out of the RichTextFormatter class, or even just changing it from a class (it's stateless...) to a function. I opted for just making the format method a static method.

Reply:
I see, yea having it as a static method makes sense!